test/Test/Fission/Web/Ping.hs

```diff
@@ -0,0 +1,34 @@
+module Test.Fission.Web.Ping (tests) where
+
+import           Servant
+
+import           Test.Tasty.Hspec
+import           Test.Fission.Prelude
+
+import           Fission.Web.Ping
+import           Fission.Web.Handler
+
+api :: Proxy API
+api = Proxy @API
+
+server :: MonadIO m => m Pong
+server = hoistServer api fromHandler (pure pong)
+
+tests :: IO TestTree
+tests =
+  testSpec "Fission.Web.Ping" do
+    describe "GET /" do
+      it "follows best practices" do
+        withServantServer api (pure server) \baseURL ->
+          serverSatisfies api baseURL defaultArgs do
```

Comment:
Few questions here:
- Are `withServantServer` and `serverSatisfies` both functions from the servant libraries?
- What does this test exactly? That it doesn't return a 5xx status?

Reply:
> Are withServantServer and serverSatisfies both functions from the servant libraries?

Yes, these are from libraries: https://hackage.haskell.org/package/servant-quickcheck-0.0.7.4/docs/Servant-QuickCheck.html

> What does this test exactly? That it doesn't return a 5xx status?

So far it's testing that:

* All valid inputs for the type defined in the `API` actually succeed (prop test)
* Has correct headers when the user is unauthorized

We can add more assertions later, but this is mainly for getting it off the ground